examples/default.yaml

```diff
@@ -117,6 +117,9 @@ additionalDisks:
 #   fsType: "ext4"
 
 ssh:
+  # Address for the host.
+  # 🟢 Builtin default: "127.0.0.1" (localhost)
+  address: null
```

Comment:
It used as "port", but the old name is kept for compatibility.

If the port is zero, then for non-localhost it will be set to 22.
For localhost (127.0.0.1), it is still assigned to a random port.

Normally it is **not** set, and only the IP address is used for SSH.